pmm-app/tests/pages/pmmSettingsPage.js

```diff
@@ -82,14 +91,16 @@ module.exports = {
     popUpTitle: "//div[@class='alert-title']",
     validationMessage: "//span[@class='error-message']",
     selectedResolution: "//span[@class='ant-slider-mark-text ant-slider-mark-text-active']",
+    sttSwitchSelector: "//strong[text()='Security Threat Tool']/parent::div/following-sibling::div/button",
+    telemetrySwitchSelector: "//strong[text()='Telemetry']/parent::div/following-sibling::div/button",
+    sttLabelTooltipLocator: "//strong[text()='Security Threat Tool']/following-sibling::span/i",
+    tooltipLocator: ".ant-tooltip-inner"
```

Comment:
I'd propose to use similar naming approach (Selector), and left locators for function that returns selectors

Reply:
yes, sorry missed that one